cli/src/pcluster/constants.py

```diff
@@ -108,6 +108,10 @@
 MIN_SLURM_NODE_PRIORITY = 1
 MAX_SLURM_NODE_PRIORITY = 2**32 - 1  # max value of uint32_t
 
+# Thresholds used to trigger a warning if Memory Based Scheduling and Flexible Instance Types are used together
+MIN_MEMORY_ABSOLUTE_DIFFERENCE = 2048
```

Comment:
I would probably put 4 GiB here.

Reply:
I considered both thresholds are "placeholders" for now.
We should have an internal discussion to decide which are meaningful values.

Anyway I'll change them to the value you suggested and then we will iterate if needed.